package.json

```diff
@@ -23,16 +23,21 @@
   "main": "./dest/prod/index.html",
   "scripts": {
     "test": "grunt test",
-    "start": "nodewebkit",
+    "dev": "grunt serve",
```

Comment:
Note: there is also the badly named `grunt play` which starts a dev server with "debug" files (unbundled, unminified). Since piskel doesn't have sourcemaps or anything, it can be quite useful.

Reply:
perhaps we should start it instead of grunt serve, if it starts faster?

"dev": "grunt play"

Reply:
It is slightly faster to start. And being able to debug real sources, get readable stacktraces etc. feels more developer friendly to me. But I am also ok with adding a second script ("dev-debug"?) later on.

As you prefer, just let me know what you decide and I'll merge this!

Reply:
lets make it use grunt play instead :) it does start faster and is better for debugging. You're right!